.travis.yml

```diff
@@ -4,19 +4,25 @@ rust:
 
 git:
   depth: 1
+  quiet: true
+
+os:
+  - linux
+  - osx
+  - windows
 
 cache:
   directories:
     - "$HOME/.cargo"
+    - "$TRAVIS_BUILD_DIR/target"
 before_cache:
-  - rm -rf "$HOME/.cargo/registry"
+  - travis_retry curl -Lf "https://github.com/lzutao/cargo-cache/releases/download/0.3.3/cargo-cache-0.3.3-${TRAVIS_OS_NAME}.tar.gz" | tar -xz --directory="$HOME/.cargo/bin"
```

Comment:
???

Reply:
There is a PR https://github.com/matthiaskrgr/cargo-cache/pull/74, but it is not merged yet.
I use cargo-cache to cleanup ~/.cargo directory before caching it.

I will remove it and move it into another PR if you prefer.

Reply:
Ping @kennytm. What is your thought on this?

Reply:
Yes please remove the cache-related changes.

Reply:
Done.